### PR TITLE
Fix notification checkbox hints

### PR DESCRIPTION
**Pull request by shurik111333**
Fix hints for notification's checkboxes. Now it changes, when user makes read\unread notification

### CODE REVIEW

File: app/assets/javascripts/notification.js.coffee [hunk 1]
```diff
@@ -6,7 +6,20 @@ $(document).on 'click', '[data-behavior~=update-is_read]', (_this) ->
       is_read: @checked
       id: @id
   notification = document.getElementById('notification' + @id)
+  check_box = document.getElementById(@id)
+  unread_link = document.getElementById("unread")
+
+  #Parse number of unread notifications
+  count = 0
+  if (/[0-9]/.exec(unread_link.text) != null)
+    count = parseInt(/[0-9]+/.exec(unread_link.text))
```
Comment (auduchinok) [inline app/assets/javascripts/notification.js.coffee]: Is there a way to store the number itself and not to parse it from a text?

**Comment by auduchinok** [comment]
Please squash your commits when ready.

File: app/assets/javascripts/notification.js.coffee (hunk 1 above)
Comment (shurik111333) [inline app/assets/javascripts/notification.js.coffee]: I think that it is possible to create .js script, that executes after main page rendering, which will initialize number of unread notification. I am working on it

**Comment by shurik111333** [comment]
Now i hide number of unread notifications if user doesn't have unread notifications. Is there right? Or it will be better to show "Непрочитанные (0)" Instead "Непрочитанные"?

File: app/views/welcome/_feed.html.erb [hunk 2]
```diff
@@ -1,13 +1,17 @@
+<%= javascript_tag do %>
+  var unread_count = <%= current_user.unread_notifications_count %>
+<% end %>
 <div class="row">
   <div class="col-md-6">
     <h1>Новости</h1>
+    <% unread_text = "Непрочитанные" + if (current_user.unread_notifications_count > 0) then " (#{current_user.unread_notifications_count})" else "" end %>
```
Comment (auduchinok) [inline app/views/welcome/_feed.html.erb]: `current_user.unread_notifications_count` can be extracted to a variable. It'll be cleaner and if there's a lot of such code it should be somewhat faster.

File: app/views/notifications/_notification.html.erb [hunk 3]
```diff
@@ -38,7 +38,7 @@
     </table>
 
 <% else %>
-  <span><%= check_box_tag "#{notification.id}", 1, notification.is_read, title: "Отметить как прочитанное", data: { behavior: "update-is_read" } %></span>
+  <span><%= check_box_tag "#{notification.id}", 1, notification.is_read, title: "Отметить как" + (notification.is_read? ? " не" : " ") + "прочитанное", data: { behavior: "update-is_read" } %></span>
```
Comment (auduchinok) [inline app/views/notifications/_notification.html.erb]: Use string interpolation.

**Comment by auduchinok** [comment]
Yeah, I like the idea of hiding the number.

**Comment by shurik111333** [comment]
Done

**Comment by auduchinok** [comment]
Please squash commits.

**Comment by shurik111333** [comment]
Done

**Pull request merged**